components/landing/landingButtons.tsx

```diff
@@ -17,7 +17,9 @@ const LandingButtons = () => {
     }, []);
 
     return (
-        <><div className={'flex flex-wrap justify-center mt-8 gap-4'}>
+        <>
+            <div className={'flex flex-wrap justify-center mt-8 gap-4'}>
+            ...
```

Comment:
lol when I said `...` I meant "everything else". Sorry for not being clear. Also make sure your commits follow the conventional commit format, yes all of them. Commits that aren't relevant to the repo history should be squashed with another one.

Reply:
did github lag? this comment like like.. an hour late to what i did

Reply:
perhaps. The other part of the review still stands tho

Reply:
yea ill squash them in a sec